app/models/events/search.rb

```diff
@@ -89,7 +99,7 @@ def today_start_of_month?
 
     def query_events_api(limit)
       GetIntoTeachingApiClient::TeachingEventsApi.new.search_teaching_events_grouped_by_type(
-        type_id: type,
+        type_ids: type_ids,
```

Comment:
This isn't currently supported by the API, but we could do this to support querying multiple event types